charger/ocpp.go

```diff
@@ -90,12 +95,12 @@ func NewOCPPFromConfig(other map[string]interface{}) (api.Charger, error) {
 	}
 
 	var currentsG func() (float64, float64, float64, error)
-	if c.hasMeasurement(types.MeasurandCurrentImport + ".L3") {
+	if c.hasMeasurement(types.MeasurandCurrentImport) {
```

Comment:
Dafür müssen auch die Templates angepasst werden?

Reply:
Ja, in der Tat. Das wird sehr übersichtlich.